test/unit/bucket/test_bucket.py

```diff
@@ -1402,6 +1402,20 @@ def test_upload_local_file_retention(self):
             self.assertEqual(retention, file_info.file_retention)
             self.assertEqual(LegalHold.ON, file_info.legal_hold)
 
+    def test_upload_bytes_cache_control(self):
+        data = b'hello world'
+        file_info = self.bucket.upload_bytes(data, 'file1', cache_control='max-age=3600')
+        self.assertTrue(isinstance(file_info, VFileVersionInfo))
+        self._check_file_contents('file1', data)
+        self.assertEqual(file_info.cache_control, 'max-age=3600')
+
+    def test_upload_file_cache_control(self):
+        data = b'hello world'
+        file_info = self.bucket.upload_bytes(data, 'file1', cache_control='max-age=3600')
```

Comment:
judging by the name of the test it should be `upload_local_file` and not `upload_bytes`. In v0 it was done well and here it wasn't, which indicates that you have not self-reviewed your PR. It's strongly recommended :)